commands/accounts/set_password.go

```diff
@@ -0,0 +1,69 @@
+// Package accounts holds commands for accounts command.
+//
+//nolint:dupl
+package accounts
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+
+	accountscli "github.com/percona/everest/pkg/accounts/cli"
+	"github.com/percona/everest/pkg/kubernetes"
+)
+
+// NewSetPwCommand returns a new set-password command.
+func NewSetPwCommand(l *zap.SugaredLogger) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "set-password",
+		Example: "everestctl accounts set-password --username user1 --password $USER_PASS",
+		Run: func(cmd *cobra.Command, args []string) { //nolint:revive
+			initSetPwViperFlags(cmd)
+
+			kubeconfigPath := viper.GetString("kubeconfig")
+			username := viper.GetString("username")
+			password := viper.GetString("password")
+
+			k, err := kubernetes.New(kubeconfigPath, l)
+			if err != nil {
+				var u *url.Error
+				if errors.As(err, &u) {
+					l.Error("Could not connect to Kubernetes. " +
+						"Make sure Kubernetes is running and is accessible from this computer/server.")
+				}
+				os.Exit(0)
+			}
+
+			cli := accountscli.New(l)
+			cli.WithAccountManager(k.Accounts())
+			if err != nil {
+				l.Error(err)
+				os.Exit(1)
+			}
```

Comment:
Looks like this block will never be executed, bc the error check above (lines 33-40) will `exit(0)` earlier, and this block checks the same `err`

Reply:
Thanks for catching, I copied over this file from `create.go` and looks like it had an issue as well. Fixed now